sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Content/ContainerRegistryContentClient.cs

```diff
@@ -671,6 +680,28 @@ private static bool ReferenceIsDigest(string reference)
             return reference.StartsWith("sha256:", StringComparison.OrdinalIgnoreCase);
         }
 
+        private static void CheckBlobSize(Response response)
```

Comment:
nit (to reuse it in manifest validation):
```suggestion
        private static void CheckContentLength(Response response)
```